practice/src/addProduct.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Page for adding food items" />
+    <meta property="og:image" content="./assets/images/logo.svg" />
+    <link rel="icon" href="./assets/images/logo.svg" type="image/x-icon" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <script
+      src="./javascripts/components/sidebar.js"
```

Comment:
I remember that you have moved this code to main.js. Please update

Reply:
In MR #38, you added a script file at the bottom. You should maintain consistency across all files.